### ISSUE TITLE
click or focus on marker

### ISSUE THREAD

**Issue by ThibautRuy** [dceejay/RedMap#37]
Hi
Would it be possible to document or enhance this node to have the possibility to focus or click on a marker after drawing. I would like to automatically point my marker and display the popup box.
Thanks
Thibaut

**Comment by dceejay** [comment]
you can automatically move the map to center the node.
Re popup - did you mean to tell it to be open when you plot the point... or to be able to tell it to open or close after placing it  (first would be part of the data  - second would be a command)

**Comment by ThibautRuy** [comment]
Hi, I manage to center the map on the marker, but I would need to automatically open the popup... by the way this node is fantastic!

**Comment by ThibautRuy** [comment]
At the moment I place the marker and center it with a command in one shot

**Comment by dceejay** [comment]
yeah - OK - I'm just thinking about if it is better as a option on the command object or just as part of the top level payload.

**Comment by ThibautRuy** [comment]
Well I have the feeling the command is quite convenient, but I could imagine use cases where you need to target multiple markers

**Comment by dceejay** [comment]
indeed - but as a standalone command it then needs to know which data to target - whereas if you supply it as part of the payload with the data it is more immediate...   so `msg.payload = {lat:... , lon:... , name:... , popup:true }` sort of thing

**Comment by ThibautRuy** [comment]
Do you think it could be possible to keep open pop-ups then ask to close them? I mean plot marker one, open pop-up, plot marker 2, open pop-up and then close pop-ups one by one?

**Comment by dceejay** [comment]
Now in 1.3.1

**Comment by ThibautRuy** [comment]
Hi,

I cannot see the update 1.3.1 in node-red. Do I need to restart my server or hasn't it been pushed yet to node-red?

Thibaut

**Comment by ThibautRuy** [comment]
Hi,

I am trying to make this work, I have upgraded to node-red-contrib-web-worldmap
1.3.1 my palette. I tried both grouping it in the json payload or separately but the pop-up does not open... It only plot the marker.


msg.payload = { name:reg, icon:"uav", iconColor:"blue", callsign:Call, id:id, operator:op, bearing:trak, model:Mdl, speed:spd, lat:lat, lon:lon, command:{layer:"OSM",lat:lat,lon:lon,zoom:3} };
msg.payload.popup = true;

**Comment by dceejay** [comment]
There is/was a slight bug in 1.3.1 for rendering svg icons... fixed in 1.3.2
but this flow should work

```
[{"id":"7e4c460a.3a4098","type":"inject","z":"745a133b.dd6dec","name":"","topic":"","payload":"","payloadType":"date","repeat":"","crontab":"","once":false,"onceDelay":0.1,"x":140,"y":1760,"wires":[["bb595f21.4ce87"]]},{"id":"bb595f21.4ce87","type":"function","z":"745a133b.dd6dec","name":"uav","func":"msg.payload = { name:\"reg\", icon:\"uav\", iconColor:\"#0000ff\", callsign:\"Papa\", id:\"1\", operator:\"Jack\", bearing:280, model:\"AB01\", speed:45, lat:51.05, lon:-1.35, command:{layer:\"OSM\",lat:51.05,lon:-1.35,zoom:16} };\nmsg.payload.popup = true;\nreturn msg;","outputs":1,"noerr":0,"x":290,"y":1760,"wires":[["8a8c27be.a66668"]]}]
```
